examples/bri-3/src/bri/transactions/agents/transactions.agent.ts

```diff
@@ -17,11 +17,17 @@ import { BpiSubjectAccount } from '../../identity/bpiSubjectAccounts/models/bpiS
 @Injectable()
 export class TransactionAgent {
   constructor(private storageAgent: TransactionStorageAgent) {}
+
   public throwIfCreateTransactionInputInvalid() {
     // TODO: This is a placeholder, we will add validation rules as we move forward with business logic implementation
     return true;
   }
 
+  public isCreateTransactionInputInvalid(): boolean {
```

Comment:
We need to be able to throw if the inpur is invalid and determine if the input is invalid without throwing?

Reply:
Yes, we throw when we work with REST API and we do not throw when inside a background task (i.e. NATS message processing).